src/pytypes/onefuzztypes/models.py

```diff
@@ -213,6 +213,8 @@ class Report(BaseModel):
     asan_log: Optional[str]
     task_id: UUID
     job_id: UUID
+    tool_name: str
+    onefuzz_version: str
```

Comment:
Is this `onefuzz_version` in the sense of versioning the crash report, or is it the the tool version? 

Should we also include `tool_version`, or just have that be part of `tool_name` if relevant?